### PR TITLE
PW88 addition

### PR DESCRIPTION
**Pull request by SmollChungus**
## Description

Addition of VIA definition for the PW88 PCB

## QMK Pull Request 

https://github.com/qmk/qmk_firmware/pull/21482

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

File: v3/smoll/pw88.json [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+                  "showIf": "{id_qmk_rgblight_effect} != 0 && {id_qmk_rgblight_effect} != 35",
+                  "label": "Color",
+                  "type": "color",
+                  "content": ["id_qmk_rgblight_color", 2, 4]
+                }
+              ]
+            }
+          ]
+        }
+      ],
```
Comment (Xelus22) [inline v3/smoll/pw88.json:56]:
```suggestion
    "menus":[ "qmk_rgblight" ],
```

**Review by Xelus22 — CHANGES_REQUESTED** [review]
Please move file to `v3/pw88/pw88.json` to reflect QMK tree

**Comment by SmollChungus** [comment]
> Please move file to `v3/pw88/pw88.json` to reflect QMK tree

My apologies. On the QMK tree, the PW88 firmware is being moved to the 'smoll' folder as per https://github.com/qmk/qmk_firmware/pull/22401. However the 'smoll' folder currently only exists on QMK's dev branch and hasn't been pulled into master yet.

Should I update this PR to reflect the current tree and make a new commit once the 'smoll' folder hits master, or temporarily close this PR? Thanks in advance!

**Comment by Xelus22** [comment]
> > Please move file to `v3/pw88/pw88.json` to reflect QMK tree
> 
> My apologies. On the QMK tree, the PW88 firmware is being moved to the 'smoll' folder as per [qmk/qmk_firmware#22401](https://github.com/qmk/qmk_firmware/pull/22401). However the 'smoll' folder currently only exists on QMK's dev branch and hasn't been pulled into master yet.
> 
> Should I update this PR to reflect the current tree and make a new commit once the 'smoll' folder hits master, or temporarily close this PR? Thanks in advance!

Could probably just wait until the breaking changes from develop go in

File: v3/smoll/pw88.json [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+          },
+          "5,12\n\n\n4,2",
+          {
+            "w": 1.5
+          },
+          "5,13\n\n\n4,2"
+        ]
+      ]
+    }
+  }
```
Comment (Cipulot) [inline v3/smoll/pw88.json:318]: Changed the KLE keys colors to match the requirements of the documentation.
Here is the KLE with fixed colors and the Design tab view to show the retention of the 2 accent keys.
image
image


```suggestion
{
  "name": "PW88",
  "vendorId": "0x5363",
  "productId": "0x8888",
  "keycodes": ["qmk_lighting"],
  "menus": ["qmk_rgblight"],
  "matrix": {"rows": 6, "cols": 17},
  "layouts": {
    "labels": [
      "Split Backspace",
      "ISO Enter",
      "Split Left Shift",
      "Split Right Shift",
      [
        "Bottom Row",
        "ANSI",
        "7U",
        "10U"
      ]
    ],
    "keymap": [
      [
        {
          "x": 2.5,
          "c": "#777777"
        },
        "0,0",
        {
          "x": 0.25,
          "c": "#cccccc"
        },
        "0,1",
        "0,2",
        "0,3",
        "0,4",
        {
          "x": 0.25,
          "c": "#aaaaaa"
        },
        "0,5",
        "0,6",
        "0,7",
        "0,8",
        {
          "x": 0.25,
          "c": "#cccccc"
        },
        "0,9",
        "0,10",
        "0,11",
        "0,12",
        {
          "x": 0.25,
          "c": "#777777"
        },
        "0,13",
        {
          "x": 0.25,
          "c": "#aaaaaa"
        },
        "0,14",
        "0,15",
        "0,16"
      ],
      [
        {
          "y": 0.25,
          "x": 2.5,
          "c": "#cccccc"
        },
        "1,0",
        "1,1",
        "1,2",
        "1,3",
        "1,4",
        "1,5",
        "1,6",
        "1,7",
        "1,8",
        "1,9",
        "1,10",
        "1,11",
        "1,12",
        {
          "c": "#aaaaaa",
          "w": 2
        },
        "1,13\n\n\n0,0",
        {
          "x": 0.25
        },
        "1,14",
        "1,15",
        "1,16",
        {
          "x": 1,
          "c": "#cccccc"
        },
        "1,13\n\n\n0,1",
        {
          "c": "#aaaaaa"
        },
        "3,14\n\n\n0,1"
      ],
      [
        {
          "x": 2.5,
          "w": 1.5
        },
        "2,0",
        {
          "c": "#cccccc"
        },
        "2,1",
        "2,2",
        "2,3",
        "2,4",
        "2,5",
        "2,6",
        "2,7",
        "2,8",
        "2,9",
        "2,10",
        "2,11",
        "2,12",
        {
          "w": 1.5
        },
        "2,13\n\n\n1,0",
        {
          "x": 0.25,
          "c": "#aaaaaa"
        },
        "2,14",
        "2,15",
        "2,16",
        {
          "x": 1.75,
          "w": 1.25,
          "h": 2,
          "w2": 1.5,
          "h2": 1,
          "x2": -0.25
        },
        "3,12\n\n\n1,1"
      ],
      [
        {
          "x": 2.5,
          "w": 1.75
        },
        "3,0",
        {
          "c": "#cccccc"
        },
        "3,1",
        "3,2",
        "3,3",
        "3,4",
        "3,5",
        "3,6",
        "3,7",
        "3,8",
        "3,9",
        "3,10",
        "3,11",
        {
          "c": "#aaaaaa",
          "w": 2.25
        },
        "3,12\n\n\n1,0",
        {
          "x": 4,
          "c": "#cccccc"
        },
        "2,13\n\n\n1,1"
      ],
      [
        {
          "c": "#aaaaaa",
          "w": 1.25
        },
        "4,0\n\n\n2,1",
        {
          "c": "#cccccc"
        },
        "4,1\n\n\n2,1",
        {
          "x": 0.25,
          "c": "#aaaaaa",
          "w": 2.25
        },
        "4,0\n\n\n2,0",
        {
          "c": "#cccccc"
        },
        "4,2",
        "4,3",
        "4,4",
        "4,5",
        "4,6",
        "4,7",
        "4,8",
        "4,9",
        "4,10",
        "4,11",
        {
          "c": "#aaaaaa",
          "w": 2.75
        },
        "4,12\n\n\n3,0",
        {
          "x": 1.25,
          "c": "#cccccc"
        },
        "4,15",
        {
          "x": 1.25,
          "c": "#aaaaaa",
          "w": 1.75
        },
        "4,12\n\n\n3,1",
        "4,13\n\n\n3,1"
      ],
      [
        {
          "x": 2.5,
          "w": 1.25
        },
        "5,0\n\n\n4,0",
        {
          "w": 1.25
        },
        "5,1\n\n\n4,0",
        {
          "w": 1.25
        },
        "5,2\n\n\n4,0",
        {
          "c": "#cccccc",
          "w": 6.25
        },
        "5,6\n\n\n4,0",
        {
          "c": "#aaaaaa",
          "w": 1.25
        },
        "5,10\n\n\n4,0",
        {
          "w": 1.25
        },
        "5,11\n\n\n4,0",
        {
          "w": 1.25
        },
        "5,12\n\n\n4,0",
        {
          "w": 1.25
        },
        "5,13\n\n\n4,0",
        {
          "x": 0.25,
          "c": "#cccccc"
        },
        "5,14",
        "5,15",
        "5,16"
      ],
      [
        {
          "y": 0.25,
          "x": 2.5,
          "c": "#aaaaaa",
          "w": 1.5
        },
        "5,0\n\n\n4,1",
        "5,1\n\n\n4,1",
        {
          "w": 1.5
        },
        "5,2\n\n\n4,1",
        {
          "c": "#cccccc",
          "w": 7
        },
        "5,6\n\n\n4,1",
        {
          "c": "#aaaaaa",
          "w": 1.5
        },
        "5,11\n\n\n4,1",
        "5,12\n\n\n4,1",
        {
          "w": 1.5
        },
        "5,13\n\n\n4,1"
      ],
      [
        {
          "x": 2.5,
          "w": 1.5
        },
        "5,0\n\n\n4,2",
        "5,1\n\n\n4,2",
        {
          "c": "#cccccc",
          "w": 10
        },
        "5,6\n\n\n4,2",
        {
          "c": "#aaaaaa"
        },
        "5,12\n\n\n4,2",
        {
          "w": 1.5
        },
        "5,13\n\n\n4,2"
      ]
    ]
  }
}
```

**Comment by Cipulot** [comment]
image

QMK Breaking Changes Q1 '24 passed and the folder structure now matches.

I've also aligned the KLE keys' color to the required one from the documentation while maintaining the accent keys proposed in the original PR.

Ping to @SmollChungus for notification. 
Cheers 🍺

**Pull request merged**